src/actions/vstudio/_vstudio.lua

```diff
@@ -340,7 +340,13 @@
 		-- Then the system libraries, which come undecorated
 		local system = config.getlinks(cfg, "system", "fullpath")
 		for i = 1, #system do
-			table.insert(links, path.appendextension(system[i], ".lib"))
+			-- Add extension if required
+			local link = system[i]
+			if link:match("[^.]+$") == link then
```

Comment:
I did a quick check, it works fine with more than one `.` in the filename. Should I add another test for this case?